llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -5265,10 +5265,62 @@ OpenMPIRBuilder::getOpenMPDefaultSimdAlign(const Triple &TargetTriple,
   return 0;
 }
 
+static void appendNontemporalVars(BasicBlock *Block,
+                                  SmallVector<Value *> &NontemporalVars) {
```

Comment:
nit
```suggestion
                                  SmallVectorImpl<Value *> &NontemporalVars) {
```

This allows small vectors with different default allocation sizes